src/nvim/ascii.h

```diff
@@ -87,4 +91,42 @@
 # define PATHSEPSTR     "/"
 #endif
 
+static inline bool ascii_iswhite(int) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
+static inline bool ascii_isdigit(int) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
+static inline bool ascii_isxdigit(int) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
+static inline bool ascii_isspace(int) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
+
+/// Checks if `c` is a space or tab character.
+static inline bool ascii_iswhite(int c)
+{
+  return c == ' ' || c == '\t';
+}
+
+/// Use our own isdigit() replacement, because on MS-Windows isdigit() returns
+/// non-zero for superscript 1.  Also avoids that isdigit() crashes for numbers
+/// below 0 and above 255.
```

Comment:
This and other are not valid `///` comments because it should look like

    /// Summary (*one* line)
    ///
    /// Extended description

I.e.

    /// Check whether character is a decimal digit
    ///
    /// Library isdigit() function is officially locale-dependent and, for 
    /// example, returns true for superscript 1 (¹) in locales where encoding 
    /// contains it in lower 8 bits. Also avoids crashes in case c is below 
    /// 0 or above 255: library functions are officially defined as accepting 
    /// only EOF and unsigned char values (otherwise it is undefined behaviour) 
    /// what may be used for some optimizations (e.g. simple `return 
    /// isdigit_table[c];`).